zipkin-storage/elasticsearch-http/src/main/java/zipkin/storage/elasticsearch/http/HttpClient.java

```diff
@@ -289,23 +322,18 @@ public void clear(String index) {
     checkState(!"RED".equalsIgnoreCase(status), "Health status is RED");
   }
 
-  private static final class IndicesHealth extends GenericResultAbstractAction {
+  private static final class IndicesHealth extends Health {
```

Comment:
yeah I switched as GenericResultAbstractAction as the Health constructor had a race condition on setting its url. That was scary to me and very dodgy code. I preferred using the base type that wasn't dodgy